tests/__init__.py

```diff
@@ -19,6 +19,20 @@
 
 __module__ = """tests"""
 
+__name__ = """tests"""
```

Comment:
🤔 hrmm as the linter complains: does this line really do anything useful?

- [ ] open issue to explore python magic variables and document resolution
- [ ] resolve this convo for this PR